content/pages/contributing.md

```diff
@@ -207,6 +204,13 @@ you wish to use it. Consult the repo-specific contributor’s guide
 for information on “Building the site”, and
 follow the steps described therein.
 
+> **Note:**
+> Repository-specific contributor's
+> information can always be accessed by navigating your web browser
+> to the appropriate Project Pythia GitHub repository,
```

Comment:
Maybe pedantic quibble, but isn't the markdown > notation supposed to be reserved for block quotes? E.g. https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf

Reply:
You could instead wrap this text in an HTML admonition, like
```
<div class="alert alert-block alert-info">
    <b>Note:</b> 
Repository-specific contributor's information can always be accessed by navigating your web browser to the appropriate Project Pythia GitHub repository... </div>
```